R/read_extract_cmh.R

```diff
@@ -5,6 +5,11 @@
 #' @export
 #'
 read_extract_cmh <- function(year, file_path = get_boxi_extract_path(year = year, type = "CMH")) {
+  # Specify years available for running
+  if (year >= 1617 & year <= 2021) {
```

Comment:
if file_path is null then return null. This is because we are going to do this in get_boxi_extract_path and do this in one place

Reply:
I've now committed `check_year_valid` to this branch which checks for the years. This means that if `is.na(file_path` then `return(NULL)` for passing through functions